### ISSUE TITLE
Add functionality to API to handle upload and retrieval of patient profiles to and from database

### ISSUE THREAD

**Issue by kristiandiana** [DSC-McMaster-U/Ocular-Disease-Identifier#26]
Description: Upon each time that a user submits images, a record of the results should be tracked in a "patient profile". This information must also be able to be retrieved by the user only (data privacy). Make a new Flask route (or 2 I guess) that would be able to handle the uploading and retrieval outlined above. We will expand upon this task later once we have authentication setup, but for now it is fine to get the functionality working for a "single user example".  Each entry should contain, but is not limited to, date & time, the image files, and the results of the classification. Must be able to handle users "signing up" to create a new entry in the database, and must be able to handle users "logging in" to add additional entries to the corresponding database. All information required by the web app for authentication will be passed through a POST request (handled by web app subteam)

Additional features: I would like the MVP to be easily demoed by users, and I expect that people don't regularly have professional fundus images lying around. With this being said, it would be good if we could initialize each user's data collection with a folder of "sample images" that they could use to test how the application works!

TLDR: make this lol and be able to test it with POSTMAN
![Image](https://github.com/user-attachments/assets/ee92f89f-a90a-4671-b4ef-8ae3f8a44f56)
